datafusion/common/src/dfschema.rs

```diff
@@ -993,18 +1059,18 @@ mod tests {
     use super::*;
     use arrow::datatypes::DataType;
 
-    #[test]
-    fn qualifier_in_name() -> Result<()> {
-        let col = Column::from_name("t1.c0");
-        let schema = DFSchema::try_from_qualified_schema("t1", &test_schema_1())?;
-        // lookup with unqualified name "t1.c0"
-        let err = schema.index_of_column(&col).unwrap_err();
-        assert_eq!(
-            err.strip_backtrace(),
-            "Schema error: No field named \"t1.c0\". Valid fields are t1.c0, t1.c1."
-        );
-        Ok(())
-    }
+    // #[test]
```

Comment:
This test is currently incorrect IMHO, it expects the error but it should run ok